src/dnssec/sign/keys/keyset.rs

```diff
@@ -99,10 +108,25 @@ impl KeySet {
         &mut self,
         pubref: String,
         privref: Option<String>,
+        algorithm: SecurityAlgorithm,
+        key_tag: u16,
         creation_ts: UnixTime,
+        available: bool,
```

Comment:
A nit: call sites would be more readable if `available` were an enum instead of a boolean flag, as currently looking at a call site one has to wonder what does `true` or `false` mean.